docs/swarm_at_scale/deploy-infra.md

```diff
@@ -0,0 +1,440 @@
+    use it to label this Engine instance.
+
+2. Set your local shell to the `keystore` Docker host.
+
+    ```bash
+    $ eval $(docker-machine env keystore)
+    ```
+3. Run <a href="https://hub.docker.com/r/progrium/consul/" target="_blank">the
+`consul` container</a>.
+
+    ```bash
+    $ docker run --restart=unless-stopped -d -p 8500:8500 -h consul progrium/consul -server -bootstrap
+    ```
+
+    The `-p` flag publishes port 8500 on the container which is where the Consul
+    server listens. The server also has several other ports exposed which you can
+    see by running `docker ps`.
+
+    ```bash
+    $ docker ps
+    CONTAINER ID        IMAGE               ...       PORTS                                                                            NAMES
+    372ffcbc96ed        progrium/consul     ...       53/tcp, 53/udp, 8300-8302/tcp, 8400/tcp, 8301-8302/udp, 0.0.0.0:8500->8500/tcp   dreamy_ptolemy
+    ```
+
+4. Use a `curl` command test the server by listing the nodes.
+
+    ```bash
+    $ curl $(docker-machine ip keystore):8500/v1/catalog/nodes
+    [{"Node":"consul","Address":"172.17.0.2"}]
+    ```
+
+
+## Task 2. Create the Swarm manager
+
+In this step, you create the Swarm manager and connect it to the `keystore`
+instance. The Swarm manager container is the heart of your Swarm cluster.
+It is responsible for receiving all Docker commands sent to the cluster, and for
+scheduling resources against the cluster. In a real-world production deployment,
+you should configure additional replica Swarm managers as secondaries for high
+availability (HA).
+
+You'll use the `--eng-opt` flag to set the `cluster-store` and
+`cluster-advertise` options to refer to the `keystore` server. These options
+support the container network you'll create later.
+
+1. Create the `manager` host.
+
+    ```bash
+    $ docker-machine create -d virtualbox --virtualbox-memory "2000" \
+    --engine-opt="label=com.function=manager" \
+    --engine-opt="cluster-store=consul://$(docker-machine ip keystore):8500" \
+    --engine-opt="cluster-advertise=eth1:2376" manager
+    ```
+
+    You also give the daemon a `manager` label.
+
+2. Set your local shell to the `manager` Docker host.
+
+    ```bash
+    $ eval $(docker-machine env manager)
+    ```
+
+3. Start the Swarm manager process.
+
+    ```bash
+    $ docker run --restart=unless-stopped -d -p 3376:2375 \
+    -v /var/lib/boot2docker:/certs:ro \
+    swarm manage --tlsverify \
+    --tlscacert=/certs/ca.pem \
+    --tlscert=/certs/server.pem \
+    --tlskey=/certs/server-key.pem \
+    consul://$(docker-machine ip keystore):8500
+    ```
+
+    This command uses the TLS certificates created for the `boot2docker.iso` or
+    the manager. This is key for the manager when it connects to other machines
+    in the cluster.
+
+4. Test your work by using displaying the Docker daemon logs from the host.
+
+    ```bash
+    $ docker-machine ssh manager
+    <-- output snipped -->
```

Comment:
Yes/ Readers will check their work against each step.  So, if we don't include the entire output of a command, we let them know we didn't so they don't worry the executed the command incorrectly.